src/schema/objects/metadata.yaml

```diff
@@ -1137,6 +1137,7 @@ Genetics:
   description: |
     An object containing information about the genetics descriptor.
   type: object
+  required_fields: [Dataset]
   properties:
     Database:
       name: Database
```

Comment:
Can you swap `DataBase` and `Dataset` below? That way the required field appears first.

Reply:
done in https://github.com/bids-standard/bids-specification/pull/1442/commits/805a77815a53dd02140e595d22765cbb12e4e13c